devicon.json

```
@@ -8656,6 +8656,35 @@
         "color": "#FB70A9",
         "aliases": []
     },
+    {
+        "name": "photonengine",
+        "altnames": [
+          "photon"
+        ],
+        "tags": [
+            "game",
+            "game-engine"
+        ],
+        "versions": {
+            "svg": [
+                "original"
+            ],
+            "font": [
+                "plain"
+            ]
+        },
```

```suggestion
        "versions": {
            "svg": [
                "original",
                "plain"
            ],
            "font": [
                "plain"
            ]
        },
```
Adds `plain` version in `svg[]`.